File: crates/dashboard-model/src/victron_controller/dashboard/from_0_1_0_decision_factor.rs


pub fn convert__decision_factor__from__0_1_0(from: &crate::victron_controller::dashboard::v0_1_0::decision_factor::DecisionFactor) -> crate::victron_controller::dashboard::decision_factor::DecisionFactor {
    crate::victron_controller::dashboard::decision_factor::DecisionFactor {
        name: from.name.clone(),
        value: from.value.clone(),
    }
}

File: web/src/model/victron_controller/dashboard/v0_2_0/from_0_1_0_decision-factor.ts
// @ts-nocheck
import {DecisionFactor as v0_2_0_DecisionFactor} from './DecisionFactor'
import {DecisionFactor as v0_1_0_DecisionFactor} from '../v0_1_0/DecisionFactor'

export function convert__decision_factor__from__0_1_0(from: v0_1_0_DecisionFactor): v0_2_0_DecisionFactor {
    return new v0_2_0_DecisionFactor (
        from.name,
        from.value
    )
}